Gemfile

```diff
@@ -11,13 +11,18 @@ group :backend, :frontend, :core, :api do
   else
     gem 'rails', ENV['RAILS_VERSION'] || '~> 6.1.0', require: false
   end
+
+  if ENV['RAILS_VERSION'] == '7.0.0'
+    gem 'awesome_nested_set', github: 'collectiveidea/awesome_nested_set'
```

Comment:
I have some concerns here. Apps that have `solidus_core` as a dependency will still be able to only get the RubyGems version of this gem (unless they add this line in their `Gemfile` explicitly) because when bundling, Bundler will only check the dependencies defined in the solidus's `gemspec`, and not this Gemfile.

Reply:
Yes, we must wait until new versions of `awesome_nested_set` and `solidus_i18n` are released.

Reply:
`solidus_i18n` release is done btw. Still waiting on awesome_nested_set release.